msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,219 @@ The following image shows the default view of the Adaptive Card when you have no
+   <br>:::image type="content" source="../../assets/images/Cards/target-width-wide.png" alt-text="Screenshot shows how adaptive card renders when the targetWidth property is set to wide."::: </br>
+   <br> Adaptive card renders when the `targetWidth` property is set to `standard` or `narrow`.</br>
+   <br>:::image type="content" source="../../assets/images/Cards/target-width-standard-narrow.png" alt-text="Screenshot shows how adaptive card renders when the targetWidth property is set to standard or narrow."::: </br>
+   <br> Adaptive card renders when the `targetWidth` property is set to `veryNarrow`.</br>
+   <br>:::image type="content" source="../../assets/images/Cards/target-width-very-narrow.png" alt-text="Screenshot shows how adaptive card renders when the targetWidth property is set to veryNarrow."::: </br>
+
+  The following code is an example of Adaptive Card designed using `targetWidth`:
+
+  ``` json
+    {
+      "type": "AdaptiveCard",
+      "body": [
+        {
+          "type": "ColumnSet",
+          "columns": [
+            {
+              "type": "Column",
+              "items": [
+                {
+                  "type": "Image", 
+                  "style": "Person", 
+                  "url": "https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg", 
+                  "size": "Small"
+                } 
+              ], 
+              "width": "auto" 
+            },
+            {
+              "type": "Column",
+              "spacing": "medium",
+              "items": [
+                {
+                  "type": "TextBlock", 
+                  "weight": "Bolder", 
+                  "text": "Matt Hidinger", 
+                  "wrap": true
+                },
+                {
+                  "type": "TextBlock", 
+                  "targetWidth": "narrow", 
+                  "spacing": "None", 
+                  "text": "Principal Program Manager", 
+                  "isSubtle": true, 
+                  "wrap": true 
+                } 
+              ], 
+              "width": "auto", 
+              "verticalContentAlignment": "center" 
+            }, 
+            { 
+              "type": "Column", 
+              "targetWidth": "atLeast:standard", 
+              "spacing": "medium", 
+              "items": [ 
+                { 
+                  "type": "TextBlock", 
+                  "text": "Principal Program Manager", 
+                  "isSubtle": true, 
+                  "wrap": true 
+                } 
+              ], 
+              "width": "stretch", 
+              "verticalContentAlignment": "center" 
+            } 
+          ] 
+        } 
+      ], 
+      "$schema": "https://adaptivecards.io/schemas/adaptive-card.json", 
+      "version": "1.5" 
+    } 
+  ```
+
+When designing your card,  you can display specific elements when the card reaches a certain width and hide the remaining elements. For example:
```

Comment:
Everything from here on is redundant, please remove/

Reply:
Updated.